ui/app/AppLayouts/Communities/controls/PermissionItem.qml

```diff
@@ -152,15 +152,60 @@ Control{
                 verticalAlignment: Text.AlignVCenter
             }
 
-            StatusListItemTag {
-                height: d.flowRowHeight
-                title: PermissionTypes.getName(root.permissionType)
-                asset.name: PermissionTypes.getIcon(root.permissionType)
-                asset.isImage: false
-                asset.bgColor: "transparent"
-                closeButtonVisible: false
-                titleText.color: Theme.palette.primaryColor1
-                titleText.font.pixelSize: d.tagTextPixelSize
+            ListModel {
+                id: tokenOwnerModel
+
+                ListElement { permissionType: PermissionTypes.Type.Owner }
+                ListElement { permissionType: PermissionTypes.Type.TokenMaster }
+                ListElement { permissionType: PermissionTypes.Type.Admin }
+            }
+
+            ListModel {
+                id: tokenMasterModel
+                ListElement { permissionType: PermissionTypes.Type.TokenMaster }
+                ListElement { permissionType: PermissionTypes.Type.Admin }
+            }
+
+            ListModel {
+                id: reqularRoleModel
+                ListElement { permissionType: PermissionTypes.Type.None }
+            }
+
+            Repeater {
+                id: rolesRepeater
+
+                model: if (root.permissionType ===  PermissionTypes.Type.Owner) {
+                           return tokenOwnerModel
+                       } else if (root.permissionType ===  PermissionTypes.Type.TokenMaster) {
+                           return tokenMasterModel
+                       } else {
+                           return reqularRoleModel
```

Comment:
the `else`s are redundant here

Reply:
Why? If I understand correctly, in order to do it without `else`, I'll need to refactor it to
```
model: {
    if (root.permissionType ===  PermissionTypes.Type.Owner) {
        return tokenOwnerModel
    } else if (root.permissionType ===  PermissionTypes.Type.TokenMaster) {
        return tokenMasterModel
    }
    return reqularRoleModel
}
```

and I was trying to escape { }

P.S. But yeh, another option is
```
model: if (root.permissionType ===  PermissionTypes.Type.Owner) 
   ? tokenOwnerModel : root.permissionType ===  PermissionTypes.Type.TokenMaster
   ? tokenMasterModel : reqularRoleModel
```
but if/else if/else looks more readable :)

Reply:
```qml
model: {
    if (root.permissionType === PermissionTypes.Type.Owner)
        return tokenOwnerModel
    if (root.permissionType === PermissionTypes.Type.TokenMaster)
        return tokenMasterModel
    return reqularRoleModel
}
```

Reply:
ah, ok, just a habit to put {} in if's 👍